assets/bootstrap.js

```diff
@@ -0,0 +1,5 @@
+import { startStimulusApp } from '@symfony/stimulus-bridge';
+import '@symfony/autoimport';
+
+// Registers Stimulus controllers from controllers.json and in the controllers/ directory
+export const app = startStimulusApp(require.context('./controllers', true, /\.(j|t)sx?$/));
```

Comment:
Are you using stimulus? (It's the 3rd time I'm asking this question). I don't think so. 
Please remove all code related to symfony/ux. It's not used in this application.